docs/modules/ROOT/pages/shipping-logs-to-amazon-elasticsearch-service.adoc

```diff
@@ -1,71 +1,102 @@
+To get a logging solution up and running, update the following domain settings. You can go back and adjust them later to suit your needs.
 
-The configuration part is done, and you should see *Loading* in your dashboard
+. In the *Domain name* field, enter a short name for your OpenSearch domain. For example, _hazelcast-cloud-logs_.
+. In the *Deployment type* section:
+** For testing purposes, select *Development and testing* as the deployment type.
+** From the *Version* list, select *Elasticsearch* > *7.10*.
+** Select *Enable compatibility mode*.
+// Is this correct or should we use the latest OpenSearch version?
+. In the *Data nodes* section, under *Availability Zones*, select *1-AZ*.
+. In the *Network* section, select *Public access* to use fine-grained access control to secure the OpenSearch domain and data. VPC Peering connections between Hazelcast Cloud clusters and OpenSearch instances are not currently supported.
+// Is this still true? 
 
-image:elasticsearch-domains-dash.png[Overview of a loading domain, including its configuration settings]
+. In the *Fine-grained access control* section, select *Create Master User* and enter a username and password for the account.
 
-After a while (~5-8 mins after), you should see it is Active and it will provide some endpoints to be used.
++
+TIP: Make a note of the master user credentials. You'll need them later to access Kibana, and to make any configuration changes to your OpenSearch domain.
++
 
-NOTE: Please store Endpoint value in a safe place to use in Hazelcast Cloud Logging Configuration step.
+. In the *Access policy* section, select *Only use fine-grained access control*.
+. Click *Create* to build your domain. You'll need to wait until your OpenSearch environment and the following URLs are created. You need both URLs to complete your logging setup.
 
-== Configure Hazelcast Cloud Logging
+** *Kibana URL*
+** *Domain endpoint*
 
-To configure logging, you can go to *Hazelcast Cluster Detail Page* > *Settings* > *Logging Configuration*
+[[configure-cloud-logging]]
+== Configure Logging Integration in Hazelcast Cloud
 
-Select *Elastic Stack (ELK)* from *Logging Technology* list.
+Configure your Hazelcast Dedicated cluster to send log files to the OpenSearch domain.
 
-image:elasticsearch-logging.png[A dropdown of logging integrations with Elastic Stack (ELK) highlighted]
+You'll need the master user credentials and domain endpoint that you created in xref:#configure-opensearch-domain[].
 
-In this Configuration page, provide *Username*, *Password*, and Url you saved in previous steps. In URL section, it will require port number, so the value will be something like `https://<domain>:443` since Elasticsearch Service provides Elasticsearch through this port not *9200*. Once you complete this step, your logging configuration will be ready in ~30 seconds. 
+. In Hazelcast Cloud, go to *Your Clusters* and select the cluster that want to integrate with OpenSearch.
```

Comment:
```suggestion
. Sign into the [.console]*link:{page-cloud-console}[{hazelcast-cloud} console]* and select the cluster that want to integrate with OpenSearch.
```

Reply:
The `[.]` is called a role and it just allows us to style the sign-in button.

Reply:
Updated as requested.